### PR TITLE
feat: add devnet 12

### PR DESCRIPTION
**Pull request by barnabasbusa**

### CODE REVIEW

File: ansible/inventories/devnet-12/host_vars/prysm-besu-1-arm.yaml [hunk 1]
```diff
@@ -0,0 +1,3 @@
+prysm_container_image: gcr.io/prylabs-dev/prysm/beacon-chain:devnet-10
```
Comment (parithosh) [inline ansible/inventories/devnet-12/host_vars/prysm-besu-1-arm.yaml:1]: remove? wrong image imo

File: terraform/devnet-12/hetzner/hetzner.tf [hunk 2]
```diff
@@ -0,0 +1,217 @@
...
+            tags            = server.labels
+            hostname        = split(".", key)[0]
+            cloud           = "hetzner"
+            region          = server.datacenter
+          }
+        }
+      )
+    }
+  )
+  filename = "../../../ansible/inventories/devnet-11/hetzner_inventory.ini"
```
Comment (parithosh) [inline terraform/devnet-12/hetzner/hetzner.tf:216]: wrong path, needs to be devnet-12

**Review by parithosh — APPROVED** [review]

File: ansible/inventories/devnet-12/host_vars/prysm-besu-1-arm.yaml (hunk 1 above)
Comment (barnabasbusa) [reply ansible/inventories/devnet-12/host_vars/prysm-besu-1-arm.yaml:1]: Nono, I wanna keep this around, to ensure that I remember. 

Its using a different image than our other prysm nodes, as its for ARM. We can leave it around, as we won't be spinning up an arm machine for prysm this time around, but its a good reminder that it needs an arm img.

File: terraform/devnet-12/hetzner/hetzner.tf (hunk 2 above)
Comment (barnabasbusa) [reply terraform/devnet-12/hetzner/hetzner.tf:216]: good catch

**Pull request merged**